### PR TITLE
feat(samples): add subordinate CA samples

### PR DESCRIPTION
**Pull request by FrodoTheTrue**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-security-private-ca/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/snippets/activate_subordinate_ca.py
Comment (dinagraves) [inline samples/snippets/activate_subordinate_ca.py:32]:
```suggestion
    Activate a subordinate Certificate Authority (CA).
```

File: samples/snippets/activate_subordinate_ca.py
Comment (dinagraves) [inline samples/snippets/activate_subordinate_ca.py:33]:
```suggestion
    *Prerequisite*: Get the Certificate Signing Resource (CSR) of the subordinate CA signed by another CA. Pass in the signed
```

File: samples/snippets/activate_subordinate_ca.py
Comment (dinagraves) [reply samples/snippets/activate_subordinate_ca.py:32]: This is just a writing suggestion -- it helps the reader to spell out the full phrase first and then indicate that you'll be using the acronym later.

File: samples/snippets/activate_subordinate_ca.py
Comment (dinagraves) [inline samples/snippets/activate_subordinate_ca.py:47]:
```suggestion
    ca_service_client = privateca_v1.CertificateAuthorityServiceClient()
```

Use snake case for python variables.  [PEP 8 Style Guide](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)

File: samples/snippets/activate_subordinate_ca.py
Comment (dinagraves) [inline samples/snippets/activate_subordinate_ca.py:61]: I wonder if it would be better to construct this outside of the function call? I feel like it's getting a little lost nested in here.

File: samples/snippets/activate_subordinate_ca.py
Comment (dinagraves) [inline samples/snippets/activate_subordinate_ca.py:77]: Why sleep here?  Can you add a comment explaining it?

File: samples/snippets/create_certificate_csr.py
Comment (dinagraves) [inline samples/snippets/create_certificate_csr.py:32]:
```suggestion
    Create a Certificate which is issued by the specified Certificate Authority (CA).
```

File: samples/snippets/create_certificate_csr.py
Comment (dinagraves) [inline samples/snippets/create_certificate_csr.py:33]:
```suggestion
    The certificate details and the public key is provided as a Certificate Signing Request (CSR).
```

File: samples/snippets/create_certificate_csr.py
Comment (dinagraves) [inline samples/snippets/create_certificate_csr.py:44]:
```suggestion
    ca_service_client = privateca_v1.CertificateAuthorityServiceClient()
```

File: samples/snippets/create_certificate_csr.py
Comment (dinagraves) [inline samples/snippets/create_certificate_csr.py:52]: `certificate = certificate =  ... `

I'm not clear what's happening here.  Is this a typo?  If not, can you add a comment?

File: samples/snippets/create_certificate_csr.py
Comment (dinagraves) [inline samples/snippets/create_certificate_csr.py:66]:
```suggestion
    print(f"Certificate created successfully: {response.name}")
```

File: samples/snippets/create_subordinate_ca.py
Comment (dinagraves) [inline samples/snippets/create_subordinate_ca.py:33]:
```suggestion
    Create Certificate Authority (CA) which is the subordinate CA in the given CA Pool.
```

File: samples/snippets/create_subordinate_ca.py
Comment (dinagraves) [inline samples/snippets/create_subordinate_ca.py:45]:
```suggestion
    ca_service_client = privateca_v1.CertificateAuthorityServiceClient()
```

File: samples/snippets/create_subordinate_ca.py
Comment (dinagraves) [inline samples/snippets/create_subordinate_ca.py:94]:
```suggestion
    print(f"Operation result: result")
```

**Review by dinagraves — CHANGES_REQUESTED** [review]
Thank you!  It's looking pretty good - make sure you replace all the camel case with snake case.  I added in some "suggestions", but you'll have to find all the references to it. 

LMK if you have any questions!

**Comment by FrodoTheTrue** [comment]
@dinagraves thanks for the review. updated!

File: samples/snippets/test_subordinate_ca.py
Comment (dinagraves) [inline samples/snippets/test_subordinate_ca.py:61]:
```suggestion
    ca_service_client = privateca_v1.CertificateAuthorityServiceClient()
```

**Review by dinagraves — APPROVED** [review]
Awesome! Looking good!  Found one more spot with the camel case that should be switched to snake case.  

Approving now - please update before you merge!  Thank you!

**Pull request merged**
